packages/@vue/cli/lib/util/ProjectPackageManager.js

```diff
@@ -76,7 +77,9 @@ class PackageManager {
     }
 
     if (!SUPPORTED_PACKAGE_MANAGERS.includes(this.bin)) {
-      throw new Error(`Unknown package manager: ${this.bin}`)
+      log()
+      warn(`Unknown package manager: ${this.bin}. It may not be well supported and we are now treating it like npm but potential errors could happen.`)
```

Comment:
I think the message here needs better phrasing… This doesn't feel intuitive enough.

Reply:
I try to rephrase the warning message. What do you think about it now. Since I'm not a native english speaker. I can really use some help here.

Reply:
I'm not a native speaker, either…
How about
```suggestion
      warn(
        `The package manager ${chalk.red(this.bin)} is not officially supported.\n` +
        `It will be treated like ${chalk.cyan('npm')}, but compatibility issues may occur.\n` +
        `See if you can use ${chalk.cyan('--registry')} instead.`
      )
```

![image](https://user-images.githubusercontent.com/3277634/65781517-4e761080-e17e-11e9-92ef-e37a18941e59.png)


/cc @NataliaTepluhina We need some help here 😂

Reply:
@sodatea I like your last suggestion, it sounds really good to me 👍